conformance-test/test-data/retryStrategyTestData.json

```diff
@@ -0,0 +1,262 @@
+            "name": "storage.objects.list",
+            "resources": [
+              "BUCKET",
+              "OBJECT"
+            ]
+          },
+          {
+            "name": "storage.serviceaccount.get",
+            "resources": []
+          }
+        ],
+        "preconditionProvided": false,
+        "expectSuccess": true
+      },
+      {
+        "id": 2,
+        "description": "conditionally idempotent retries when precondition is present",
+        "cases": [
+          {
+            "instructions": [
+              "return-503",
+              "return-503"
+            ]
+          }
+        ],
+        "methods": [
+          {
+            "name": "storage.buckets.patch",
+            "resources": [
+              "BUCKET"
+            ]
+          },
+          {
+            "name": "storage.buckets.setIamPolicy",
+            "resources": [
+              "BUCKET"
+            ]
+          },
+          {
+            "name": "storage.buckets.update",
+            "resources": [
+              "BUCKET"
+            ]
+          },
+          {
+            "name": "storage.hmacKey.update",
+            "resources": []
+          },
+          {
+            "name": "storage.objects.compose",
+            "resources": [
+              "BUCKET",
+              "OBJECT"
+            ]
+          },
+          {
+            "name": "storage.objects.copy",
+            "resources": [
+              "BUCKET",
+              "OBJECT"
+            ]
+          },
+          {
+            "name": "storage.objects.delete",
+            "resources": [
+              "BUCKET",
+              "OBJECT"
+            ]
+          },
+          {
+            "name": "storage.objects.insert",
+            "resources": [
+              "BUCKET"
+            ]
+          },
+          {
+            "name": "storage.objects.patch",
+            "resources": [
+              "BUCKET",
+              "OBJECT"
+            ]
+          },
+          {
+            "name": "storage.objects.rewrite",
+            "resources": [
+              "BUCKET",
+              "OBJECT"
+            ]
+          },
+          {
+            "name": "storage.objects.update",
+            "resources": [
+              "BUCKET",
+              "OBJECT"
+            ]
+          }
+        ],
+        "preconditionProvided": true,
+        "expectSuccess": true
+      },
+      {
+        "id": 3,
+        "description": "conditionally idempotent no retries when precondition is absent",
+        "cases": [
+          {
+            "instructions": []
+          }
+        ],
+        "methods": [],
+        "preconditionProvided": false,
+        "expectSuccess": false
+      },
+      {
+        "id": 4,
+        "description": "non idempotent",
+        "cases": [
+          {
+            "instructions": []
+          }
+        ],
+        "methods": [],
+        "preconditionProvided": false,
+        "expectSuccess": false
+      }
+    ]
+  }
```

Comment:
Probably get caught by the linter but this file is missing a new line.

Reply:
`v4SignedUrl.json` also doesn't have a new line. I think this is fine?

Reply:
or is new line the correct thing to do